docs/reference-architectures/virtual-machines-linux/multi-vm.md

```diff
@@ -70,7 +70,10 @@ To route traffic to a specific VM, use NAT rules. For example, to enable RDP to
 
 ### Storage account recommendations
 
-Create separate Azure storage accounts for each VM to hold the virtual hard disks (VHDs), in order to avoid hitting the input/output operations per second [(IOPS) limits][vm-disk-limits] for storage accounts. 
+Create separate Azure storage accounts for each VM to hold the virtual hard disks (VHDs), in order to avoid hitting the input/output operations per second [(IOPS) limits][vm-disk-limits] for storage accounts.
```

Comment:
Add "If you are not using managed disks" and move below the note? As per single VM doc.